README.md

```diff
@@ -15,7 +15,7 @@ This table shows a list of boards that are supported.
 
 Board                               |  Connectivity     | Storage for credentials and FW candidate | Notes
 ------------------------------------| ------------------| ------------------------| --------------
-Cypress `CYTFM_064B0S2_4343W`       | Wi-Fi             | Internal flash for credentials + external flash for FW candidate | To use `mbed-cloud-client-example` with the `CYTFM_064B0S2_4343W` board, please check out the [cytfm-064b0s2-4343w](../cytfm-064b0s2-4343w/TARGET_CYTFM_064B0S2_4343W/README.md) branch.
+Cypress `CYTFM_064B0S2_4343W`       | Wi-Fi             | Internal flash for credentials + external flash for FW candidate | To use `mbed-cloud-client-example` with the `CYTFM_064B0S2_4343W` board, please check out the `cytfm-064b0s2-4343w` branch and see the [*Running Device Management Client example on the CYTFM_064B0S2_4343W* document](../cytfm-064b0s2-4343w/TARGET_CYTFM_064B0S2_4343W/README.md).
```

Comment:
```suggestion
Cypress `CYTFM_064B0S2_4343W`       | Wi-Fi             | Internal flash for credentials + external flash for FW candidate | To use `mbed-os-example-pelion` with the `CYTFM_064B0S2_4343W` board, please check out the `cytfm-064b0s2-4343w` branch and see the [*Running PDMC example on the CYTFM_064B0S2_4343W* document](../cytfm-064b0s2-4343w/TARGET_CYTFM_064B0S2_4343W/README.md).
```
This line becomes too long I think.
Or,
To run this example on the `CYTFM_064B0S2_4343W`, please check out the `cytfm-064b0s2-4343w` branch and see [*CYTFM_064B0S2_4343W/README.md document](../cytfm-064b0s2-4343w/TARGET_CYTFM_064B0S2_4343W/README.md).